.github/workflows/coverage.yml

```diff
@@ -20,20 +25,48 @@ jobs:
   Coveralls:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      # checkout comparison branch
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.comparison_branch }}
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+          registry-url: 'https://registry.npmjs.org'
+      - run: npm ci --ignore-scripts
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - run: npm rebuild
+      # get base coverage
+      - run: ${{ inputs.test_script }}
+      - uses: yext/slapshot-reusable-workflows/get-coverage-percent@dev/coverage-workflow-prescan
+        id: base
+      # checkout current branch
+      - uses: actions/checkout@v3
       - name: Use Node.js 16.x
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v3
         with:
           node-version: 16.x
           registry-url: 'https://registry.npmjs.org'
-      # Skip post-install scripts here, as a malicious
-      # script could steal the NPM_TOKEN.
       - run: npm ci --ignore-scripts
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
-      # `npm rebuild` will run all those post-install scripts for us.
       - run: npm rebuild
+      # get current coverage
       - run: ${{ inputs.test_script }}
+      - uses: yext/slapshot-reusable-workflows/get-coverage-percent@dev/coverage-workflow-prescan
+        id: current
+      - name: Check if percent is too low
+        run: |
+          percentIsTooLow=$(echo ${{ steps.base.outputs.coverage-percent }} ${{ steps.current.outputs.coverage-percent }} |
+            awk '{print ($1 - $2 > 2) }')
+          echo $percentIsTooLow
```

Comment:
can we include which branch correspond to which percentage?

Reply:
Are you thinking of just including an echo somewhere? can definitely do that

Reply:
oh I think you are echoing the percentanges here no? yea that would be nice

Reply:
no this isn't echoing the percentage, it's echoing the percentIsTooLow "boolean"

Reply:
this is what a run looks like https://github.com/yext/answers-react-components/runs/6908003012?check_suite_focus=true

Reply:
sweet!

Reply:
it's parallelized now! here is the test run https://github.com/yext/answers-react-components/runs/6908712378?check_suite_focus=true